docs/content/moose_tools/memory_logger.md

```diff
@@ -0,0 +1,242 @@
+# Memory Logger
+Memory Logger is a simple tool for gathering information about a running process. Such things as memory usage, stdout and stack traces can be sampled, either on a single process or on multiple nodes in a job scheduling environment. Currently PBS is the only job scheduling system supported.
+
+## Basic usage
+In its simplest use case scenario, you encapsulate the command you would normally run with out Memory Logger as an argument to Memory Logger:
+```text
+memory_logger.py --run "mpiexec -n 2 /absolute/path/to/moose_test-opt -i simple_diffusion.i -r 5"
+```
+!!! Important
+    Always use absolute paths between those --run double quotes. Memory Logger has no knowledge on paths like: '../../  or ~/some/path'
+
+Memory Logger will launch your process as normal, and begin to track the two processes that launched. When Memory Logger detects that the process/s has exited (for good or for ill), it will explain where the log was saved. You can then instruct Memory Logger to open this file, and display its information either using Gnuplot (--plot), or by formatting the information and outputting it to stdout (--read).
```

Comment:
You can probably change this to say 'matplotlib' and then just refer to "plot" or "graph" at the other locations.